_posts/2015-05-23-photoacoustic-penetration.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: "Hearing with light: photoacoustic radiology improved"
```

Comment:
True, but the transducers do the listening and convert it to visual signals. Should I make that clear? Or should I change the title to something along the lines of your previous comment (https://github.com/apostilb/apostilb.github.io/pull/37#discussion_r30946942)?